frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.spec.ts

```diff
@@ -91,12 +91,14 @@ describe(`Testing uninstall of ${testOperator.name} Operator`, () => {
   it(`attempts to uninstall the Operator, shows 'Error uninstalling Operator' alert`, () => {
     // invalidate the request so operator doesn't get uninstalled, as opposed to
     // letting request go thru unchanged and mocking the response
-    cy.intercept('DELETE', '/api/kubernetes/apis/operators.coreos.com/*', (req) => {
-      // examples:
-      // .../api/kubernetes/apis/operators.coreos.com/v1alpha1/namespaces/test-mpnsw/subscriptions/datagrid-foobar
-      // .../api/kubernetes/apis/operators.coreos.com/v1alpha1/namespaces/test-mpnsw/clusterserviceversions/datagrid-operator.v8.2.0-foobar
-      req.url = `${req.url}-foobar`;
-    }).as('deleteOperatorSubscriptionAndCSV');
+    cy.intercept(
+      'DELETE',
+      '/api/kubernetes/apis/operators.coreos.com/*/namespaces/*/*/*',
```

Comment:
I'm debating if this much specificity in the URL is better or worse. Was there a problem with `/api/kubernetes/apis/operators.coreos.com/*` or did you just want something more precise?

Reply:
I was having issues with the wildcard, seemed like dir breaks (`/`) need to be specified!?

Reply:
@dtaylor113 We should be able to use `/api/kubernetes/apis/operators.coreos.com/**`

https://docs.cypress.io/api/commands/intercept#Usage

Reply:
Note that it needs two `**`